acceptance/localcluster/localcluster.go

```diff
@@ -332,6 +332,7 @@ func (l *Cluster) processEvent(e dockerclient.EventOrError) bool {
 	// An event on any other container is unexpected. Die.
 	select {
 	case <-l.stopper:
+	case <-monitorStopper:
```

Comment:
The point is that calling Stop twice will panic because it will close monitorStopper twice. None of the current tests do that but it is intended based on the comments.